lms/static/sass/shared/_course_object.scss

```diff
@@ -265,5 +314,6 @@
         margin-right: 0;
```

Comment:
Not your doing, but RTL Support (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#right-to-left-rtl-support) is needed here.